charts/tidb-operator/values.yaml

```diff
@@ -217,14 +217,18 @@ admissionWebhook:
     ## apiservice config
     ## refer to https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/#contacting-the-extension-apiserver
     insecureSkipTLSVerify: true
-    ## The key and cert for `tidb-admission-webook.<Release Namespace>` Service.
+    ## The tls key and cert for `tidb-admission-webook.<Release Namespace>.svc` Service in Secret.
     ## If insecureSkipTLSVerify is true, this would be ignored.
-    cert: ""
-    key: ""
+    ## You can create the tls secret by:
+    ## kubectl create secret tls tidb-admission-webhook-certs  --cert=<path-to-cert> --key=<path-to-key> --namespace <release-namespace>
+    tlsSecret: ""
+    ## The caBundle for the webhook apiservice, you could get it by the secret you created previously:
+    ## kubectl get secret tidb-admission-webhook-certs -o=jsonpath='{.data.tls\.crt}'
+    caBundle: ""
   ## certProvider indicate the key and cert for the webhook configuration to communicate with `kubernetes.default` service.
   ## If your kube-apiserver's version >= 1.13.0, you can leave cabundle empty and the kube-apiserver
   ## would trust the roots on the apiserver.
   ## refer to https://github.com/kubernetes/api/blob/master/admissionregistration/v1/types.go#L529
   ## or you can get the cabundle by:
-  ## kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}'
+  ## kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 | tr -d '\n'
```

Comment:
` | base64 | tr -d '\n'` can be changed to ` | base64 -w0`?

Reply:
I think we could only base64 is necessary, the options for tr is different for GNU and BSD here